copter/source/docs/traditional-helicopter-autotune.rst

```diff
@@ -0,0 +1,197 @@
+|                      | +---------+-------------------+ |
+|                      | | 5       | Roll & Yaw        | |
+|                      | +---------+-------------------+ |
+|                      | | 6       | Pitch & Yaw       | |
+|                      | +---------+-------------------+ |
+|                      | | 7       | Roll, Pitch & Yaw | |
+|                      | +---------+-------------------+ |
++----------------------+---------------------------------+
+
+:ref:`Maximum Response Gain<AUTOTUNE_GN_MAX>`
+---------------------------------------------
+
+Specifies the maximum response gain to be used to tune the Rate D, Rate P and Angle P gains.  The response gain is the output of the response divided by the input.  It is recommended that 1.8 to 2.0 be used for pitch and roll axes as these axes are typically more lightly damped.  It is recommended that 1.0 to 1.4 be used for the yaw axis.
+
+
+:ref:`Minimum Test Frequency<AUTOTUNE_FRQ_MIN>`
+-----------------------------------------------
+
+Specifies the minimum frequency in radians per second used during the dwell or frequency sweeps.  For frequency sweeps, this will be the starting frequency.
+
+
+:ref:`Maximum Test Frequency<AUTOTUNE_FRQ_MAX>`
+-----------------------------------------------
+
+Specifies the maximum frequency in radians per second used during the dwell or frequency sweeps.  For frequency sweeps, this will be the ending frequency.
+
+
+:ref:`Velocity P Gain<AUTOTUNE_VELXY_P>`
+----------------------------------------
+
+Specifies P gain for velocity feedback.  This aids the autotune in maintaining aircraft position during the frequency sweeps and dwells.  Keep this at 0.1 unless the aircraft is drifting more than 10 meters during the dwell and frequency sweeps.  It only affects position holding while the aircraft is oscillating during these tests.  In between the oscillations, it may drift.  This gain will not help with that.
+
+
+Preparing for Autotune
+======================
+
+Ensure you complete all of the items in the :ref:`Preparing for Tuning<traditional-helicopter-tuning-preparing>` wiki.
+
+Transmitter Setup
+-----------------
+
+Be sure to put the Autotune flight mode as one of the flight modes on your transmitter flight mode switch.  You don’t want to be reaching for the GCS to switch out of the autotune if your heli is not behaving properly.  You want to be able to switch modes instantly.
+
+
+Autotune Flights
+================
+Suggested Tuning Sequence
+-------------------------
+
+In any axis, it is recommended to conduct the feedforward gain tuning first, then the Rate D and Rate P gain tuning, and lastly the Angle P tuning.  Don't try to tune every axis in one flight, it could take up to 10-15 min to tune one axis.  Here is a suggested flights and tuning sequence. Yaw feedforward tuning is not needed for conventional helicopters.  The yaw feedforward gain can be set to zero.  It may be needed for dual helicopters though.
```

Comment:
```suggestion
In any axis, it is recommended to conduct the feedforward gain tuning first, then the Rate D and Rate P gain tuning, and lastly the Angle P tuning.  Don't try to tune every axis in one flight, it could take up to 10-15 min to tune one axis.  Here is a suggested flights and tuning sequence. Conventional helicopters do not need Yaw feedforward tuning, and the gain can be set to zero.  Dual helicopters may require Yaw feedforward tuning.
```

Reply:
done. thanks!